src/main/resources/application-local.yml

```diff
@@ -0,0 +1,8 @@
+faf:
+  environment: LOCAL
+
+logging:
+  level:
+    org.springframework.security: INFO
+    com.faforever: INFO
+    org.thymeleaf: TRACE
```

Comment:
Do we want to leave this at Trace?

Reply:
:shrug: Local environment. I can put it back to info